ses/index.cgi

```diff
@@ -42,12 +42,13 @@ sub main {
     elsif ( $message_type eq 'Notification' ) {
         my $notification = decode_json_wrapper( $message->{Message} ) // return;
 
-        my $notification_type = $notification->{notificationType} // '';
+        my $notification_type = $notification->{eventType} // $notification->{notificationType} // '';
         if ( $notification_type eq '' ) {
             my $keys = join ', ', keys %$notification;
-            WARN("No notificationType in notification (keys: $keys)");
+            WARN("No notification type in message (keys: $keys)");
+            respond( 200 => 'OK' );
```

Comment:
is 200 correct here?

Reply:
hrm, 400 is probably better, and won't trigger a resend.